### PR TITLE
Unblock 0.11.5 RC packaging

### PR DESCRIPTION
**Pull request by rnpridgeon**
Couldn't come up with a good solution in time but this should unblock us for the time being

### CODE REVIEW

File: tools/test-manylinux.sh [hunk 1]
```diff
@@ -55,8 +55,16 @@ function run_single_in_docker {
...
 
     # Install modules
+
+    # TODO: revisit to avoid hardcoding dependencies
+    pip install "futures;python_version=='2.7'"
+    pip install "enum34;python_version=='2.7'"
+
     pip install confluent_kafka --no-index -f $wheelhouse
+
+    # https://github.com/pypa/pip/issues/5247
```
Comment (edenhill) [inline tools/test-manylinux.sh]: Can you write in the comment why the double installs are needed, providing the URL for background info.

**Comment by rnpridgeon** [comment]
Thanks @edenhill , updated with a brief description

File: tools/test-manylinux.sh [hunk 2]
```diff
@@ -62,6 +62,7 @@ function run_single_in_docker {
 
     pip install confluent_kafka --no-index -f $wheelhouse
 
+    # Pytest relies on a new version of 6; later versions of pip fail to remove older versions gracefully
```
Comment (edenhill) [inline tools/test-manylinux.sh]: "new version of 6"?  Do you mean six?

Comment (rnpridgeon) [reply tools/test-manylinux.sh]: yes...

File: tools/test-manylinux.sh [hunk 3]
```diff
@@ -55,8 +55,17 @@ function run_single_in_docker {
...
+    # TODO: revisit to avoid hardcoding dependencies
+    pip install "futures;python_version=='2.7'"
+    pip install "enum34;python_version=='2.7'"
+
     pip install confluent_kafka --no-index -f $wheelhouse
+
+    # Pytest relies on a new version of six; later versions of pip fail to remove older versions gracefully
+    # https://github.com/pypa/pip/issues/5247
     pip install pytest
+    pip install pytest --ignore-installed six
```
Comment (edenhill) [inline tools/test-manylinux.sh]: do we need to install pytest twice?

Comment (rnpridgeon) [reply tools/test-manylinux.sh]: how embarrassing... that is likely what caused my issues with six to begin with

**Comment by rnpridgeon** [comment]
@edenhill , fixed

**Pull request merged**